packages/mobile/src/components/details-tile/MetadataRow.tsx

```diff
@@ -0,0 +1,18 @@
+import type { PropsWithChildren } from 'react'
+
+import { Flex, Text } from '@audius/harmony-native'
+
+type MetadataRowProps = PropsWithChildren<{
+  label: string
+}>
+
+export const MetadataRow = ({ label, children }: MetadataRowProps) => {
+  return (
+    <Flex direction='row' alignItems='center' key={label} gap='xs'>
```

Comment:
what about using <text> for everything? personally i kinda like that cause we can use spaces as the gap, but down for this also

Reply:
sry using what for everything?